### PR TITLE
[vPDQ]  Add vpdq/CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by tigerchensh**
Summary
---------

Add more documentation for vPDQ!

### CODE REVIEW

**Review by Dcallies — DISMISSED** [review]
Can you split the CONTRIBUTING.MD changes into their own PR, or move the functional changes to a new PR?

File: vpdq/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+To cmake CPP binary files:
+```
+cd vpdq/cpp/
+mkdir -p build
+cd build
+cmake ..
+make
+```
+To locally install python-binding library:
```
Comment (BarrettOlson) [inline vpdq/CONTRIBUTING.md:46]: aside: have you determined a python version requirements?

File: vpdq/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+the main project. In most cases you’ll be looking to compare against the Main
+branch, but there are instances where you’re making changes that you want to go
+into a specific branch. Make sure when submitting your PR that you choose the
+right destination branch.
+
+Once you’ve submitted a PR you're waiting on us. In most cases we like to have
+a core developer get some eyes on the code and the feature to make sure
+there’s no general issues. They might require you to go back and make some more
+changes (simply edit your local branch and push to the branch associated with
+the PR; it will get updated automagically!) .
```
Comment (BarrettOlson) [inline vpdq/CONTRIBUTING.md:81]: hmm, I have the same concern/question with these details being copied from https://github.com/facebook/ThreatExchange/blob/main/python-threatexchange/CONTRIBUTING.md as well.

**Review by BarrettOlson — CHANGES_REQUESTED** [review]
Thanks for add this!

I think we can probably reduce down the focus of this CONTRIBUTING.md to be vPDQ specific and how releases to PyPI work with a link to the creating a PR docs/details.

**Comment by tigerchensh** [comment]
Thanks for the review. Have replaced the duplicated text in pytx and vpdq with a link to the main contributing.md.

File: vpdq/CONTRIBUTING.md (hunk 1 above)
Comment (tigerchensh) [reply vpdq/CONTRIBUTING.md:25]: I think both 3.8 and 3.9 work, but I haven't tested for other versions.

File: python-threatexchange/CONTRIBUTING.md [hunk 3]
```diff
@@ -1,26 +1,5 @@
...
-your changes in your branch and test thoroughly. If this is a large feature you
-can push your branch to your fork often. This allows you to request feedback for
-how things are progressing instead of dumping a large code change all at once.
-
-When making commits to your branch, make sure you write [well-formed][wf] commit
-messages and update documentation accordingly (see the next section).
-
-[wf]: https://github.com/erlang/otp/wiki/Writing-good-commit-messages
-
+Please see [CONTRIBUTING](../CONTRIBUTING.md) for how to make contributions
```
Comment (BarrettOlson) [inline python-threatexchange/CONTRIBUTING.md:2]: nit wording: ...for how to ~make contributions~ develop locally and open PRs

nit nit: Also this file still has a copy of the `## Submit a PR` & `## Documenting Changes` sections that you added to root's CONTRIBUTING

File: vpdq/CONTRIBUTING.md (hunk 1 above)
Comment (BarrettOlson) [reply vpdq/CONTRIBUTING.md:25]: Do you think it is something worth specifying here or in the READ.md ?

File: vpdq/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
+# Contributing
+Please see [CONTRIBUTING](../CONTRIBUTING.md) for how to make contributions
```
Comment (BarrettOlson) [inline vpdq/CONTRIBUTING.md:2]: .

**Review by BarrettOlson — APPROVED** [review]
LGTM with a few more nits inline. Thanks

**Comment by tigerchensh** [comment]
Thanks for the review. I will also a section in vpdq/README for python-binding since it currently doesn't have one.

**Pull request merged**